all/pom.xml

```diff
@@ -60,6 +60,11 @@
                             <type>zip</type>
                             <target>/apps/saas-aem-module-packages/application/install</target>
                         </embedded>
+                        <embedded>
+                            <groupId>com.valtech.aemsaas</groupId>
```

Comment:
let's use groupId `com.valtech.aem` and artifactid artifactId `saas.api` do you agree?

Reply:
yup, let's :)